### DISCUSSION TITLE
[Weekly Review Report]: December 2-6

### DISCUSSION THREAD

**Discussion by susancoort** [wikipathways/wikipathways-help#148] [curation-reports]
### Recent changes

 * WP5123: Updated community tag IEM
 * WP4830: Updated community tag Rare Diseases
 * WP5053: Updated community tag Rare Diseases
 * 

### Under construction

 * WP5468: Added the Needs work tag. All nodes are annotated. However it isn't a pathway, but a list of cytokines involved in pro-inflammatory signaling. 

### New pathways

 * WP5498: Removed the test pathway tag since deletion was already requested. 

### Deleted Pathways

 * ... 

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

### Additional notes

Search on classis website works.

**Comment by khanspers** [comment]
Thank you @susancoort! For [WP5468](https://classic.wikipathways.org/index.php/Pathway:WP5468), I believe this is being used for a curation project, maybe @marvinm2 can comment?
